### DISCUSSION TITLE
Worldwide overview map in .osm file format

### DISCUSSION THREAD

**Discussion by jogurtas** [osmandapp/OsmAnd#14123] [Maps Related]
Is it possible to download [worldwide overview map](http://download.osmand.net/download.php?file=World_basemap_2.obf.zip&event=2&osmandver=OsmAndIOs+4.1.1) in `.osm` file format?

**Comment by vshcherb** [comment]
Nope it's not possible, it consists of multiple OSM files actually. You can try to find something useful here https://builder.osmand.net/basemap/
